db/migrate/20190225185919_add_pdf_template_reference_to_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddPdfTemplateReferenceToClaims < ActiveRecord::Migration[5.2]
```

Comment:
The template reference to use is stored in the claim in case the pdf needs to be generated again at any point